packages/action-link-test/package.js

```diff
@@ -0,0 +1,18 @@
+Package.describe({
+	name: 'rocketchat:action-link-test',
+	version: '0.0.1',
+	summary: 'action links test',
+	git: '',
+});
+
+Package.onUse(function(api) {
+	api.versionsFrom('1.0');
+
+	api.use('ecmascript');
+	api.use('templating');
+	api.use('rocketchat:lib');
+	
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2673573723/issues/source?bid=3199634&fileBranchId=3333125#l14)